app/src/main/java/org/mozilla/fenix/compose/LightDarkPreview.kt

```diff
@@ -0,0 +1,8 @@
+package org.mozilla.fenix.compose
+
+import android.content.res.Configuration
+import androidx.compose.ui.tooling.preview.Preview
+
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO)
+annotation class LightDarkPreview
```

Comment:
let's move this initial change into its own commit, so we'll ultimately land two commits with this PR